src/wp-admin/options-general.php

```diff
@@ -199,7 +199,7 @@ class="<?php echo esc_attr( $classes_for_button ); ?>"
 	<p class="description">
 		<?php
 			/* translators: %s: Site Icon size in pixels. */
-			printf( __( 'Site Icons are what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. They should be square and at least %s pixels.' ), '<code>512 &times; 512</code>' );
+			printf( __( 'The site icon is what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. It should be square and at least %s pixels.' ), '<code>512 &times; 512</code>' );
```

Comment:
The feature name has always been capitalized and is also capitalized in the GB implementation as of last week
 
```suggestion
			printf( __( 'The Site Icon is what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. It should be square and at least %s pixels.' ), '<code>512 &times; 512</code>' );
```